src/Connection/Internal/Http2ConnectionProcessor.php

```diff
@@ -711,6 +714,7 @@ public function handlePriority(int $streamId, int $parentId, int $weight): void
         }
 
         $stream = $this->streams[$streamId];
+        $stream->resetInactivityWatcher();
```

Comment:
Should priority changes really reset the inactivity watcher?

Reply:
Nah, probably not. Removed.